BackofficeBundle/Resources/translations/interface.en.yml

```diff
@@ -2,3 +2,11 @@ open_orchestra_backoffice:
     table:
         keywords:
             label: Label
+    dashboard:
+        last_node_title: My latest pages
+        last_content_title: My latest contents
+        node_draft_title: My pages not published
+        content_draft_title: My contents not published
+        widget:
+            list:
+              no_entities: No records found
```

Comment:
@alavieille : to be consistent with the french translation you should have here "No element to display" or "Pas d'enregistrement trouvé" in the french version.